java/src/jmri/jmrit/operations/setup/JmritOperationsSetupBundle.properties

```diff
@@ -74,7 +74,8 @@ AddLogo             = Add Logo
 RemoveLogo          = Remove Logo
 Reset               = Reset
 
-#The following keys must exactly match their English translation, used as tags in the xml file.
+#The following keys must not be modified or backwards compatibility will be broken, used as tags in the xml file.
```

Comment:
If these should not be translated, they should be constants in the Setup.java file. Even with your comments, this is an invitation to failure.